lms/templates/instructor/instructor_dashboard_2/student_admin.html

```diff
@@ -113,6 +116,46 @@
   </div>
 %endif
 
+%if section_data['access']['instructor']:
+  <hr>
+  <div class='course-specific-container action-type-container'>
+    <h2>${_("Submit External Grades")}</h2>
+    <p>
+      ${_("To import grades from an assignment outside of the Lagunita platform, "
+      "ask the Course Operations team to set up external grading for your course. "
+      "Then download the grade submission CSV with the button below and fill in the missing fields. "
+      "Once you have saved your changes, select an assignment you are importing from the drop-down. "
+      "Finally, upload and submit your grades." )}
```

Comment:
I can remove it

On Tuesday, June 2, 2015, Kevin Luo <notifications@github.com> wrote:

> In lms/templates/instructor/instructor_dashboard_2/student_admin.html
> <https://github.com/Stanford-Online/edx-platform/pull/296#discussion_r31595873>
> :
>
> > @@ -113,6 +116,46 @@
> >    </div>
> >  %endif
> >
> > +%if section_data['access']['instructor']:
> > +  <hr>
> > +  <div class='course-specific-container action-type-container'>
> > +    <h2>${_("Submit External Grades")}</h2>
> > +    <p>
> > +      ${_("To import grades from an assignment outside of the Lagunita platform, "
> > +      "ask the Course Operations team to set up external grading for your course. "
> > +      "Then download the grade submission CSV with the button below and fill in the missing fields. "
> > +      "Once you have saved your changes, select an assignment you are importing from the drop-down. "
> > +      "Finally, upload and submit your grades." )}
>
> Is the extra space after the end quote supposed to be there?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Stanford-Online/edx-platform/pull/296/files#r31595873>
> .
>